WordPress/src/main/res/values/strings.xml

```diff
@@ -2146,5 +2142,13 @@
     <string name="site_creation_tagline_hint">Site tagline (optional)</string>
     <string name="site_creation_tagline_helper">The tagline is a short line of text shown right below the title in most themes, and act as site metadata on search engines.</string>
     <string name="site_creation_empty_site_title">Please enter a site title</string>
+    <string name="site_creation_creating_laying_foundation">Laying site foundation…</string>
+    <string name="site_creation_creating_fetching_info">Retrieving site information…</string>
+    <string name="site_creation_creating_configuring_content">Configuring site content…</string>
+    <string name="site_creation_creating_configuring_theme">Configuring site style…</string>
+    <string name="site_creation_creating_preparing_frontend">Preparing frontend…</string>
+    <string name="site_creation_creating_failed">Something went wrong...</string>
```

Comment:
Using `…` and `...` can cause interface (e.g. text wrap) and accessibility (e.g. reads as "3 periods") issues.  We can replace those with `&#8230;` instead.

Reply:
Oops, good find, missed that! Done with e0ee5d3.